polaris-for-vscode/package.json

```diff
@@ -30,23 +30,27 @@
   ],
   "main": "./dist/client.js",
   "scripts": {
-    "build": "tsc -b",
-    "dev": "tsc -b -w",
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
     "lint": "TIMING=1 eslint --cache .",
     "clean": "rm -rf .turbo node_modules dist *.tsbuildinfo"
   },
   "dependencies": {
-    "@shopify/polaris-tokens": "*",
-    "vscode-languageclient": "^7.0.0",
-    "vscode-languageserver": "^7.0.0",
+    "vscode-languageclient": "^8.0.1",
+    "vscode-languageserver": "^8.0.1",
     "vscode-languageserver-textdocument": "^1.0.4"
   },
   "devDependencies": {
+    "@rollup/plugin-babel": "^5.3.1",
+    "@rollup/plugin-commonjs": "^21.1.0",
+    "@rollup/plugin-node-resolve": "^13.2.1",
+    "@shopify/polaris-tokens": "*",
```

Comment:
@samrose3 I'm not familiar with the VS Code `client/server` architecture, but is `@shopify/polaris-tokens` not a regular dependency since it's used in `server.ts` (and has access to the `node_modules`)?

Reply:
Normally yes, this would be in the `dependencies` list rather than `devDependencies`.  But, when using `vsce package`, there is an error not being able to find `@shopify/polaris-tokens` likely due to not being aware of it as part of the Yarn workspace. When using `vsce package --yarn`, the `node_modules` are omitted from being included. When trying to bundle `node_modules` using Rollup, we run into circular dependencies that causes failures in the `semver` package.

All that to say, this is the best route around all these edge-case issues. Maybe a lot of this can be resolved after migrating to pnpm 🤷 we'll see